### DISCUSSION TITLE
是不是与ublock orign不兼容

### DISCUSSION THREAD

**Discussion by Areire** [the1812/Bilibili-Evolved#4854] [提问 / Q&A]
每次使用，开启ublock origin总是会出现误删的情况，而把Bilibili-Evolved关闭后则只屏蔽广告。或者把ublock origin关闭，则也正常。求把兼容问题改善一下。

**Comment by the1812** [comment]
兼容的啊, 有什么误删吗

**Reply by Areire** [reply]
就是ublock origin会误删一些首页的普通视频

**Reply by the1812** [reply]
那可以把 ublock origin 排除一下 b 站, 脚本也支持屏蔽广告
